src/benchmarklib/tpch/tpch_benchmark_item_runner.cpp

```diff
@@ -68,10 +68,12 @@ std::string TPCHBenchmarkItemRunner::_calculate_date(boost::gregorian::date date
 }
 
 void TPCHBenchmarkItemRunner::on_tables_loaded() {
-  // Make sure that sort order, indexes, and constraints have made it all the way up to here
+  // Make sure that clustering, indexes, and constraints have made it all the way up to here
   const auto orders_table = Hyrise::get().storage_manager.get_table("orders");
   const auto first_chunk = orders_table->get_chunk(ChunkID{0});
-  Assert(!first_chunk->individually_sorted_by().empty(), "Sorting information was lost");
+  if (_config->clustering_configuration == ClusteringConfiguration::TPCHPruning) {
+    Assert(!first_chunk->individually_sorted_by().empty(), "Sorting information was lost");
+  }
```

Comment:
If we cannot find a nice way to pass this information down here, we could either remove the assert or deduce the sorting by looking at the data itself.